file/staticFile/main.go

```diff
@@ -17,13 +17,39 @@
 package main
 
 import (
+	"context"
+	"time"
+
+	"github.com/cloudwego/hertz/pkg/app"
 	"github.com/cloudwego/hertz/pkg/app/server"
+	"github.com/cloudwego/hertz/pkg/protocol/consts"
 )
 
 func main() {
 	h := server.Default(server.WithHostPorts("127.0.0.1:8080"))
 
+	h.Static("/static", "./")
+
 	h.StaticFile("/main", "./main.go")
 
+	indexNames := []string{"1.txt", "2.txt"}
+	h.StaticFS("/static1", &app.FS{
+		Root:        "./",
+		PathRewrite: app.NewPathSlashesStripper(1),
+		PathNotFound: func(_ context.Context, ctx *app.RequestContext) {
```

Comment:
Need to add some comments to help users understand

Reply:
+1, maybe add them to readme.
Explain: what & how for this fields.

Reply:
ok

Reply:
![image](https://user-images.githubusercontent.com/97824201/233052387-1eb9911c-c2b9-43e7-8a9f-08588179ea74.png)
No classification and title introduce